bundle_service/processing/process_bundle.py

```diff
@@ -109,14 +109,31 @@ async def process(rows: List[dict], project_id: str, access_token: str) -> list[
             temp_file.close()
 
         if files_written:
-            subprocess.run(["jsonschemagraph", "gen-dir", "iceberg/schemas/graph", f"{temp_dir}", f"{temp_dir}/OUT", "--project_id", f"{project_id}", "--gzip_files"])
-            res = bulk_load(await _get_grip_service_name(), await _get_grip_graph_name(), project_id, f"{temp_dir}/OUT", logs, access_token)
-            if int(res[0]["status"]) != 200:
-                server_errors.append(res[0]["message"])
+            program, project = project_id.split("-")
+            project_str_dict = f'{{"auth_resource_path":"/programs/{program}/projects/{project}"}}'
+            print(f"Using project: {project_str_dict}")
+            result = subprocess.run(
+                ["jsonschemagraph", "gen-dir", "iceberg/schemas/graph", f"{temp_dir}", f"{temp_dir}/OUT", "--extraArgs", project_str_dict, "--gzip_files"],
+                capture_output=True,
+                text=True,
+                check=False
+            )
+            if result.returncode == 0:
+                print("jsonschemagraph ran successfully.")
+                res = bulk_load(await _get_grip_service_name(), await _get_grip_graph_name(), project_id, f"{temp_dir}/OUT", logs, access_token)
+                if int(res[0]["status"]) != 200:
+                    server_errors.append(res[0]["message"])
+            else:
+                print(f"jsonschemagraph failed with exit code: {result.returncode}")
+                print("Stdout:")
```

Comment:
just confirming, the stdout gets logged to the FHIR server pod logs?

Reply:
Any error logs will come as a https response message. Any more in depth logs can be checked by checking the fhir server logs in k8s.